Pipfile

```diff
@@ -41,3 +41,9 @@ test = "pytest -vvs"
 
 [pipenv]
 allow_prereleases = true
+
+[resolver]
+python_full_version = "3.7.13"
+python_version = "3.7"
+finder_python = "3.7"
+os_name = "win32"
```

Comment:
How do teams that work on multiple OS can specify multiple OSs here?

Reply:
This is a minimal patch, so it doesn't provide a way to target multiple versions -- I went down that path initially but it involves changes in the packaging markers code, and it got too complicated for a first pass.